qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -497,16 +497,74 @@ def from_operator(obj, atol=None, rtol=None):
         return SparsePauliOp(paulis, coeffs, copy=False)
 
     @staticmethod
-    def from_list(obj):
-        """Construct from a list [(pauli_str, coeffs)]"""
+    def from_list(obj, num_qubits=None):
+        """Construct from a list of Pauli strings and coefficients.
+
+        Each list element can be
+        * a tuple of a Pauli string and a coefficient
+        * a triple of a local, non-trivial Pauli string, an index where to apply it, and a
+          coefficient
```

Comment:
Do you still prefer it if we don't allow mixing? I was slightly biased towards `(partial Pauli, indices, coefficient)` since it's a bit less nesting and the `from_list` already includes a lot of nested lists/tuples 🙂

Reply:
Personally I prefer the nested for because the partial Pauli and the indices are tightly bound, and together have the same meaning as a full Pauli.  However, you should optimise for your uses not mine, because I don't use these things, so I don't know what's most convenient for you, and my perspectives on these things tend to be a little bit more around theoretical data structures than actual usability.